packages/yoroi-extension/app/themes/skins/FormFieldOwnSkin.js

```diff
@@ -76,8 +76,8 @@ export const FormFieldOwnSkin = class extends React.Component<Props, State> {
             {(this.props.type === 'password') ? (
               <button tabIndex="-1" type="button" onClick={this.showPassword}>
                 {isPasswordShown
-                  ? <PasswordSvg />
-                  : <PasswordHiddenSvg />}
+                  ? <PasswordHiddenSvg />
+                  : <PasswordSvg />}
```

Comment:
@AhmedIbrahim336 , icons are in wrong order, you are returning "password hidden" icon when password is shown.